dataprep/eda/missing/compute.py

```diff
@@ -84,8 +84,8 @@ def missing_spectrum(df: dd.DataFrame, bins: int, ncols: int) -> Intermediate:
     cols = df.columns[:ncols]
     ncols = len(cols)
     nrows = len(df)
-    chunk_size = len(df) // num_bins
-
+    #Bug137 resolved
+    chunk_size = len(df) # num_bins
```

Comment:
Is this related to issue 137? Seems It changes the meaning of the code. E.g., if len(df) = 50 and num_bins = 10, len(df) // num_bins would be 5. But here it would be 50.